services/masterdata/src/lib/core/services/masterdata-helper.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from "@angular/core";
+import { TranslateService } from "@ngx-translate/core";
+
+@Injectable()
+export class MasterdataHelperService {
```

Comment:
Tip: using keywords like Helper is an indication of a class that don't have a high cohesion ([have too many responsibilities](https://medium.com/clarityhub/low-coupling-high-cohesion-3610e35ac4a6)). Its better to narrow it down to its specific function. E.g.: in this case you could use as class name MetadataRenderer, which covers the functionality a lot more than helper

Reply:
@Johanh-od MetadataRenderer or MasterdataRenderer ?

Reply:
Based on the comment at line 11 I guess its rendering metadata, so MetadataRenderer should be the name.

Reply:
@Johanh-od I've decided on MasterdataRendererService. I've also already another help method in this class: titleCase!